### PR TITLE
[ES] Update `web/css/::selection`

### PR DESCRIPTION
**Pull request by agatemosu**

### CODE REVIEW

**Comment by agatemosu** [comment]
`{{EmbedLiveSample()}}` doesn't work with translations? None of the translations of (at least) this page are working.

File: files/es/web/css/_doublecolon_selection/index.md [hunk 1]
```diff
@@ -1,56 +1,97 @@
...
 
-## Especificaciones
+### Resultado
+
+{{EmbedLiveSample('Ejemplos')}}
+
+## Preocupaciones de accesibilidad
 
-El `::selection` CSS pseudo-elemento fue redactado selector CSS nivel 3 pero quitado antes que alcanzara el estatus recomendado. Apesar de esto es simplemente en algunos Navegadores, que probablemente tendran soporte a este.
+**No anules los estilos del texto seleccionado por razones puramente estéticas** — los usuarios pueden personalizarlos para satisfacer sus necesidades. Para las personas que experimentan problemas cognitivos o que tienen menos conocimientos tecnológicos, los cambios inesperados en los estilos de selección pueden perjudicar su comprensión de la funcionalidad.
```
Comment (Graywolf9) [inline files/es/web/css/_doublecolon_selection/index.md:77]: Improve traduction

```suggestion
**No sobreescribas los estilos del texto seleccionado por razones puramente estéticas** — los usuarios pueden personalizarlos para satisfacer sus necesidades. Para las personas que experimentan problemas cognitivos o que tienen menos conocimientos tecnológicos, los cambios inesperados en los estilos de selección pueden perjudicar su comprensión de la funcionalidad.
```

File: files/es/web/css/_doublecolon_selection/index.md [hunk 2]
```diff
@@ -1,56 +1,97 @@
...
 
-En estos momentos, el seudo elemento CSS `::selection` no esta en ninguna especificación.
+Si se anula, es importante asegurarse de que la **relación de contraste** entre el texto y los colores de fondo de la selección sea lo suficientemente alta como para que las personas con problemas de visión puedan leerla.
```
Comment (Graywolf9) [inline files/es/web/css/_doublecolon_selection/index.md:79]:
```suggestion
Si se sobreescribe, es importante asegurarse de que la **relación de contraste** entre el texto y los colores de fondo de la selección sea lo suficientemente alta como para que las personas con problemas de visión puedan leerla.
```

File: files/es/web/css/_doublecolon_selection/index.md [hunk 3]
```diff
@@ -1,56 +1,97 @@
...
+
+La relación de contraste de color se encuentra comparando la luminosidad del texto seleccionado y los colores de fondo del texto seleccionado. Para cumplir con las [pautas de accesibilidad del contenido web (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) actuales, el contenido del texto debe tener una relación de contraste de **4.5:1**, o 3:1 para texto más largo como encabezamiengtos. (WCAG define texto grande entre `18.66px` y `24px` con [negrita](/es/docs/Web/CSS/font-weight), o `24px` o superior).
```
Comment (Graywolf9) [inline files/es/web/css/_doublecolon_selection/index.md:81]: typo and improve traduction

```suggestion
La relación de contraste de color se encuentra comparando la luminosidad del texto seleccionado y los colores de fondo del texto seleccionado. Para cumplir con las [pautas de accesibilidad del contenido web (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) actuales, el contenido del texto debe tener una relación de contraste de **4.5:1**, o 3:1 para texto más largo como encabezados. (WCAG define texto grande entre `18.66px` y `24px` con [negrita](/es/docs/Web/CSS/font-weight), o `24px` o superior).
```

File: files/es/web/css/_doublecolon_selection/index.md [hunk 4]
```diff
@@ -1,56 +1,97 @@
...
+
+- [WebAIM: Comprobador de contraste de color](https://webaim.org/resources/contrastchecker/)
```
Comment (Graywolf9) [inline files/es/web/css/_doublecolon_selection/index.md:83]: Improve traduction

```suggestion
- [WebAIM: Comprobar contraste de color](https://webaim.org/resources/contrastchecker/)
```

File: files/es/web/css/_doublecolon_selection/index.md [hunk 5]
```diff
@@ -1,56 +1,97 @@
...
+- [MDN Comprensión de las WCAG, pauta 1.4 explicaciones](/es/docs/Web/Accessibility/Understanding_WCAG/Perceivable#pauta_1.4_facilitar_a_los_usuarios_ver_y_oír_el_contenido_incluyendo_la_separación_entre_el_primer_plano_y_el_fondo)
+- [Understanding Success Criterion 1.4.3 | W3C Understanding WCAG 2.0](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)
```
Comment (Graywolf9) [inline files/es/web/css/_doublecolon_selection/index.md:85]: Missing traduction

```suggestion
- [Comprender el Criterio de Conformidad 1.4.3 | W3C: comprensión de las WCAG 2.0](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)
```

File: files/es/web/css/_doublecolon_selection/index.md [hunk 6]
```diff
@@ -1,56 +1,97 @@
 ---
 title: "::selection"
 slug: Web/CSS/::selection
+l10n:
+  sourceCommit: c6894b701b2297a7523d07855b47605c06f29c91
```
Comment (Graywolf9) [inline files/es/web/css/_doublecolon_selection/index.md:5]: This is the commit hash from the `en-US` file in the version we are taking to translate

```suggestion
  sourceCommit: 5fea7c9593f5e4b4ef13ec65064acf1eabf01e4e
```

**Review by Graywolf9 — APPROVED** [review]
Thank you so much for all your work and help! I made some minnor correction that you can check.

Thank you so much again and feel free to keep contributing

Welcome to MDN

File: files/es/web/css/_doublecolon_selection/index.md (hunk 3 above)
Comment (agatemosu) [reply files/es/web/css/_doublecolon_selection/index.md]: ._.

File: files/es/web/css/_doublecolon_selection/index.md (hunk 6 above)
Comment (agatemosu) [reply files/es/web/css/_doublecolon_selection/index.md]: I don't know what happened lol

**Pull request merged**